### ISSUE TITLE
Data normalization in cell painting gallery

### ISSUE THREAD

**Issue by tilenkranjc** [cytodata/cytodata-hackathon-2018#12]
There are multiple versions of single cell features in cell painting gallery. Could you describe which processing tasks were done on them? I figured out the `_augmented`, which is features + metadata. Where could I find information about processing steps for the other files? Are they the same as in the source papers? 

For example: `_normalized` - what kind of normalization was used?
`_feature_select` - what methodology was used?

Basically, any information that I could use to reproduce the processing steps used on the datasets.

**Comment by ErinWeisbart** [comment]
Profiles are made using scripts from [PyCytominer](https://github.com/cytomining/pycytominer) and the various outputs are described there.

The [Cell Painting Gallery README](https://github.com/broadinstitute/cellpainting-gallery/blob/main/README.md) links to any existing repositories used to create profiles which will contain the exact PyCytominer configurations used.
